From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Tue, 30 Jul 2024 15:50:12 -0400
Subject: ci: avoid another date-dependent failure

Set the day before setting the month to avoid
failure when running the test on the 31st day of a month

(cherry picked from commit 20b00d3947c65136775c47e65db674ad6aeb7b52)

Forwarded: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7535
Comment: included in gtk main after 4.15.4
---
 testsuite/gtk/notify.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/gtk/notify.c b/testsuite/gtk/notify.c
index 3e8f3ab..7ed5b92 100644
--- a/testsuite/gtk/notify.c
+++ b/testsuite/gtk/notify.c
@@ -580,8 +580,8 @@ G_GNUC_END_IGNORE_DEPRECATIONS
       /* avoid day 30 and 31, since they don't exist in February */
       instance = g_object_new (type,
                                "year", 1984,
-                               "month", 10,
                                "day", 05,
+                               "month", 10,
                                NULL);
     }
   /* special casing for singletons */
